packages/components/src/tools-panel/tools-panel-header/component.js

```diff
@@ -27,35 +41,87 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 		return null;
 	}
 
+	const defaultItems = Object.entries( menuItems?.default || {} );
+	const optionalItems = Object.entries( menuItems?.optional || {} );
+
 	return (
 		<h2 { ...headerProps } ref={ forwardedRef }>
 			{ labelText }
 			{ hasMenuItems && (
-				<DropdownMenu icon={ moreVertical } label={ labelText }>
+				<DropdownMenu
+					icon={ moreVertical }
+					label={ labelText }
+					menuProps={ { style: { minWidth: '200px' } } }
```

Comment:
Ideally we'd like to avoid applying CSS via the inline `style` prop — the preferred option would be to pass them via a `className`

Reply:
I was under the impression I needed to avoid adding a stylesheet for components and didn't see how I could use styled-components within the `DropdownMenu` here. I'll revisit this as I work through the other suggestions. Thanks.

Reply:
You could define styles in the `styles.js` file, and create a className by using the `cx` function in the hook.js file

<details>

<summary>Something like this (expand):</summary>

```diff
diff --git a/packages/components/src/tools-panel/styles.js b/packages/components/src/tools-panel/styles.js
index d9c7a985c0..c4b644fe9c 100644
--- a/packages/components/src/tools-panel/styles.js
+++ b/packages/components/src/tools-panel/styles.js
@@ -46,6 +46,10 @@ export const ToolsPanelHeader = css`
 	}
 `;
 
+export const DropdownMenu = css`
+	min-width: 300px;
+`;
+
 export const ToolsPanelItem = css`
 	grid-column: span 2;
 
diff --git a/packages/components/src/tools-panel/tools-panel-header/component.js b/packages/components/src/tools-panel/tools-panel-header/component.js
index 73af46f2be..5ff8304318 100644
--- a/packages/components/src/tools-panel/tools-panel-header/component.js
+++ b/packages/components/src/tools-panel/tools-panel-header/component.js
@@ -34,6 +34,7 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 		menuItems,
 		resetAll,
 		toggleItem,
+		dropdownMenuClassName,
 		...headerProps
 	} = useToolsPanelHeader( props );
 
@@ -51,7 +52,7 @@ const ToolsPanelHeader = ( props, forwardedRef ) => {
 				<DropdownMenu
 					icon={ moreVertical }
 					label={ labelText }
-					menuProps={ { style: { minWidth: '200px' } } }
+					menuProps={ { className: dropdownMenuClassName } }
 				>
 					{ ( { onClose } ) => (
 						<>
diff --git a/packages/components/src/tools-panel/tools-panel-header/hook.js b/packages/components/src/tools-panel/tools-panel-header/hook.js
index c903c24725..73f7fe19fe 100644
--- a/packages/components/src/tools-panel/tools-panel-header/hook.js
+++ b/packages/components/src/tools-panel/tools-panel-header/hook.js
@@ -22,6 +22,10 @@ export function useToolsPanelHeader( props ) {
 		return cx( styles.ToolsPanelHeader, className );
 	}, [ className ] );
 
+	const dropdownMenuClassName = useMemo( () => {
+		return cx( styles.DropdownMenu );
+	}, [] );
+
 	const { menuItems, hasMenuItems } = useToolsPanelContext();
 
 	return {
@@ -29,5 +33,6 @@ export function useToolsPanelHeader( props ) {
 		hasMenuItems,
 		menuItems,
 		className: classes,
+		dropdownMenuClassName,
 	};
 }
```
</details>

Reply:
Brilliant, thanks! That certainly makes it obvious how it all fits together. 🙇 

I'll add those suggestions tomorrow.

Reply:
✅ Done.